react/tsconfig.json

```diff
@@ -12,7 +12,6 @@
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noUnusedLocals": true,
```

Comment:
Why remove that?

Reply:
the `@typescript-eslint/no-unused-vars` already covers that